src/components/screens/Overviews.js

```diff
@@ -236,7 +236,8 @@ export const Overviews = ({ navigation, route }) => {
     if (
       query === QUERY_TYPES.POINTS_OF_INTEREST &&
       showMap &&
-      initialFilter === FILTER_TYPES.LIST
+      initialFilter === FILTER_TYPES.LIST &&
+      switchBetweenListAndMap == SWITCH_BETWEEN_LIST_AND_MAP.BOTTOM_FLOATING_BUTTON
```

Comment:
what does the initialFilter and switchBetweenListAndMap do and where is this visible?

Reply:
what they "do" is the wrong question. both are variables holding a value at  the time of the condition. if these values match the ones in the condition, the condition gets true, otherwise false.